src/ext/resources/legacygraphqlapi/schema.graphqls

```diff
@@ -1195,6 +1253,43 @@ input ParkingFilter {
     select: [ParkingFilterOperation!]
 }
 
+input PlanDebugSettings {
+    debugItineraryFilter: Boolean
+}
+
+input PlanLocation @oneOf {
+    coordinate: Coordinate
+    """TODO should place string be a scalar?"""
+    place: String
+}
+
+input PlanLocations {
```

Comment:
Ì would inline these, origin and destination do not share any properties, do they?

Reply:
Unlike many of the other parameters these are both required and important, so I feel it is ok to give them more "space", I would stuff away the less important fields.